src/helpers/constants/error.messages.js

```diff
@@ -23,5 +23,6 @@ export default {
   gender: 'The gender should only contain alphanumeric characters',
   phoneNumber: 'The phone number is required',
   bio: 'The bio is required',
-  rating: 'you can not go beyond 5 as a rating number, Thanks'
+  rating: 'you can not go beyond 5 as a rating number, Thanks',
```

Comment:
Tell me the use of this maybe we can come . up with a better message

Reply:
that message has been written by @MCFrank16